runtime/vm/createramclass.cpp

```diff
@@ -1954,8 +1954,7 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 
 			U_8 *preloadClassName = J9UTF8_DATA(preloadClassNameUtf8);
 			U_16 preloadClassLength = J9UTF8_LENGTH(preloadClassNameUtf8);
-
-			internalFindClassUTF8(currentThread, preloadClassName, preloadClassLength, classLoader, classPreloadFlags);
+			internalFindClassUTF8(currentThread, preloadClassName, preloadClassLength, classLoader, J9_FINDCLASS_FLAG_EXISTING_ONLY);
```

Comment:
`J9_FINDCLASS_FLAG_EXISTING_ONLY` won't trigger a class to be loaded. It only searches for the class in the existing loaded classes in the class table. The flag should be `classPreloadFlags & ~J9_FINDCLASS_FLAG_THROW_ON_FAIL`.

Reply:
For null restricted field types (or Q types), any class loading exception should still be thrown. This block of code handling preload attribute here can be moved after the following while loop that exams each field (probably right before `done:`).

Reply:
Thanks. I have updated my change.